lang/tr/texts/discover.html

```diff
@@ -9,6 +9,7 @@ <h1>Open Food Facts</h1>
 		<h2 class="subheader">Özgür gıda ürünleri veritabanı.</h2>
 		<blockquote style="border-left:none;border-right: 1px solid #DDDDDD;">Tell me what you eat and I will tell you what you are. (Bana ne yediğini söyle, ben de sana kim olduğunu söyleyeyim.*)
 		(Bana ne yediğini söyle, ben de sana kim olduğunu söyleyeyim.*)
+		(Bana ne yediğini söyle, ben de sana kim olduğunu söyleyeyim.*)
```

Comment:
3rd time's a charm? 😄  Is this a bug in the translations or in Crowdin, which introduces the redundant translations?